support/defaults.go

```diff
@@ -8,4 +8,5 @@ const (
 	CodeFlareSDKVersion = "v0.20.2"
 	RayVersion          = "2.35.0"
 	RayImage            = "quay.io/modh/ray:2.35.0-py39-cu121"
+	RayAMDGpuImage      = "quay.io/rhoai/ray:2.35.0-py39-rocm61-torch24-fa26"
```

Comment:
This rocm-ray image `quay.io/rhoai/ray:2.35.0-py39-rocm61-torch24-fa26` is torch compatible.
IMO, we should use base Ray AMD/ROCm image"2.35.0-py39-rocm61"  built with konflux
`quay.io/modh/ray:2.35.0-py39-rocm61`